### PR TITLE
Home page branch

### PR DESCRIPTION
**Pull request by ahmed-salaah**

### CODE REVIEW

File: Module/shellXamarin.Module.ElLa3eba/Templates/HomeTemplateSelector.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+namespace shellXamarin.Module.ElLa3eba.Templates
+{
+    public class HomeTemplateSelector : DataTemplateSelector
+    {
+        public DataTemplate TeamRankingTemplate { get; set; }
+        public DataTemplate ChooseRoleTemplate { get; set; }
+        public DataTemplate FeedsTemplate { get; set; }
+        public DataTemplate CommingGameTemplate { get; set; }
+        public DataTemplate DefaultTemplate { get; set; }
+        //public DataTemplate CheckTemplate { get; set; }
```
Comment (mostafasany) [inline Module/shellXamarin.Module.ElLa3eba/Templates/HomeTemplateSelector.cs]: ??

File: Module/shellXamarin.Module.Common/Assets/Mocks/en/menu.json [hunk 2]
```diff
@@ -62,5 +62,15 @@
     "badge": 1,
```
Comment (mostafasany) [inline Module/shellXamarin.Module.Common/Assets/Mocks/en/menu.json]: what about english and Arabic

File: Module/shellXamarin.Module.Common/Styles/AppStyles.xaml [hunk 3]
```diff
@@ -1,102 +1,225 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 
-<ResourceDictionary MergedWith="theme:DarkTheme" xmlns="http://xamarin.com/schemas/2014/forms" xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml" xmlns:theme="clr-namespace:shellXamarin.Module.Common.Themes;assembly=shellXamarin.Module.Common" x:Class="shellXamarin.Module.Common.Styles.AppStyles">
```
Comment (mostafasany) [inline Module/shellXamarin.Module.Common/Styles/AppStyles.xaml]: do we need to change appStyles?

File: Module/shellXamarin.Module.ElLa3eba/Models/HomeModel.cs [hunk 4]
```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace shellXamarin.Module.ElLa3eba.Models
+{
+
+    public enum TypeEnum
```
Comment (mostafasany) [inline Module/shellXamarin.Module.ElLa3eba/Models/HomeModel.cs]: 👍

File: Module/shellXamarin.Module.ElLa3eba/Models/HomeModel.cs [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    {
+        Ranking,
+        Role,
+        Feeds,
+        CommingGame
+    }
+    public class HomeModel
```
Comment (mostafasany) [inline Module/shellXamarin.Module.ElLa3eba/Models/HomeModel.cs]: I have some concerns here , let talk later

File: Module/shellXamarin.Module.Common/Styles/AppStyles.xaml (hunk 3 above)
Comment (ahmed-salaah) [reply Module/shellXamarin.Module.Common/Styles/AppStyles.xaml]: I changed it to LightTheme only

**Pull request merged**